### PR TITLE
Update landing page

### PR DESCRIPTION
**Pull request by enyia21**
#### 🔗 Pivotal Tracker ticket
#187042038

#### ✍️ Description
1. Match ggc-app's desktop and mobile appearances to the designs in figma.  

#### 📷 Design reference
[ Figma ]( https://www.figma.com/file/1oRFY7p6tdGogBJk7Ugle8/IL-CCAP-Full-Flow-(WIP)?type=design&node-id=1727-19181&mode=design&t=83snes9SLqPUe6jZ-4 )

#### ✅ Completion tasks

- [ ] Added relevant tests
- [ ] Meets acceptance criteria
- [ ] Added testing instructions to ticket

### CODE REVIEW

**Comment by sree-cfa** [comment]
looks like this section in the design doesnt quite match the changes in this branch
Screenshot 2024-02-14 at 7 57 00 AM

File: src/main/resources/templates/index.html [hunk 1]
```diff
@@ -30,22 +30,25 @@ <h1 class="spacing-below-35 spacing-above-5" th:text="#{index.title}"></h1>
...
       </div>
     </section>
     <section class="slab slab--white">
       <div class="grid">
-        <div class="index-child-care-image-container">
-            <img src="/assets/images/seal-of-illinois.png" alt="" style="width: 110px; height: 110px;">
-            <img src="/assets/images/IDHS-logo.png" alt="">
+        <div class="index-child-care-image-container width-one-fourth">
+            <img src="/assets/images/seal-of-illinois.png" alt="" style="width: 141px; height: 141px;">
+            <img src="/assets/images/IDHS-logo.png" alt="" style="width: 141px; height: auto">
```
Comment (coltborg) [inline src/main/resources/templates/index.html:40]: There's about double spacing in the design:

2024-02-14 at 11 59 23@2x

2024-02-14 at 12 00 36@2x

Comment (enyia21) [reply src/main/resources/templates/index.html]: I think that I have the spacing issues correct.   not the space between the state seal in the idhs symbol had been doubled and the size of the idhs symbol is has been increased to match design

File: src/main/resources/static/assets/css/custom.css [hunk 2]
```diff
@@ -130,14 +122,18 @@ a, .button--link {
...
+  margin-right: 13px;
+}
 .eligibility-table {
   width: 100%;
   border-collapse: collapse;
 }
 
 .eligibility-table th {
-  font-weight: normal;
+  font-weight: bold;
```
Comment (coltborg) [inline src/main/resources/static/assets/css/custom.css:136]: Adding `bold` here effects all `eligibility-table`'s on the site. 

Yes the headers need to be bold on the `homepage`, but they shouldn't be `bold` on the `onboarding-getting-started`.

Homepage (yes needs `bold` headers):
2024-02-14 at 12 48 53@2x

onboarding-getting-started on this branch:
2024-02-14 at 12 52 12@2x


onboarding-getting-started figma designs:
2024-02-14 at 12 51 28@2x

File: src/main/resources/templates/index.html [hunk 3]
```diff
@@ -30,30 +30,33 @@ <h1 class="spacing-below-35 spacing-above-5" th:text="#{index.title}"></h1>
...
     </section>
     <section class="slab slab--light-purple">
       <div class="grid">
         <h2 th:text="#{general.eligibility-table.title}"></h2>
         <table class="eligibility-table">
           <thead>
-          <tr>
+          <tr class="color--idhs-base-blue bold">
             <th scope="col" th:text="#{general.eligibility-table.column.header-1}"></th>
             <th scope="col" th:text="#{general.eligibility-table.column.header-2}"></th>
```
Comment (coltborg) [inline src/main/resources/templates/index.html:61]: The `.bold` class has to be on a `<th>` to take effect, see it's [definition in custom.css for reference, aka `th.bold` means the class has to be on the element](https://github.com/codeforamerica/il-gcc-form-flow/blob/284e7c4e70a24b3e56f8b93333adbc5d99301d2c/src/main/resources/static/assets/css/custom.css#L146-L149).

Also looks like the design color changed from `black` to the `idhs-base-blue`. I like they way you did it better with the utility class. So in `custom.css` the `.bold` needs to be modified to not set the color:

```diff
/* in custom.css:146 */
.eligibility-table th.bold {
  font-weight: bold;
- color: var(--black);
}
```

```suggestion
          <tr>
            <th class="color--idhs-base-blue bold" scope="col" th:text="#{general.eligibility-table.column.header-1}"></th>
            <th class="color--idhs-base-blue bold" scope="col" th:text="#{general.eligibility-table.column.header-2}"></th>
```

Comment (enyia21) [reply src/main/resources/templates/index.html:61]: Why does the `bold` change the appearance of the font when the page is rendered?  I am going to change it but I'm still curious

**Review by coltborg — APPROVED** [review]
lgtm;

**Pull request merged**